requirements/doc.in

```diff
@@ -7,3 +7,4 @@ edx_sphinx_theme          # edX theme for Sphinx output
 readme_renderer           # Validates README.rst for usage on PyPI
 Sphinx                    # Documentation builder
 sphinxcontrib-napoleon    # Google Style docstring support for sphinx
+twine
```

Comment:
You'll need to run `make upgrade` to update the `*.txt` files for these changes in the `*.in` files to take effect.

Reply:
@jmbowman we ignore those https://github.com/edx/web-fragments/blob/master/.gitignore#L77.

Reply:
That's definitely a bug, we need the fully-pinned requirements files for reproducible test runs.  Otherwise a new package release can break tests when we're just trying to make a trivial unrelated change.